janitor/functions.py

```diff
@@ -55,6 +57,78 @@
 )
 
 
+def get_symbol(symbol: str) -> str:
```

Comment:
I think this function might be better suited inside the `finance.py` submodule. Would you be open to moving it there?

Reply:
**Action item:** Please chime in on your thoughts about moving the function to the `finance` submodule, and if the answer is affirmative, please go ahead.